t_fault_injection/test_fault_injection_base.py

```diff
@@ -109,6 +125,67 @@ def teardown_fail_function_test():
         cmd = remote.client.run_cmd(cmd)
 
 
+class TestDtb(TestFailFunctionBase):
+    tempesta = {
+        "config": """
+            listen 80;
+            listen 443 proto=h2,https;
+
+            cache 2;
+            cache_fulfill * *;
+
+            tls_certificate ${tempesta_workdir}/tempesta.crt;
+            tls_certificate_key ${tempesta_workdir}/tempesta.key;
+            tls_match_any_server_name;
+
+            server ${server_ip}:8000 conns_n=1;
+
+            frang_limits {
+                http_strict_host_checking false;
+            }
+        """
+    }
+
+    clients = [
+        {
+            "id": "h2load",
+            "type": "external",
+            "binary": "h2load",
+            "ssl": True,
+            "cmd_args": (
+                " https://${tempesta_ip}"
+                f" --clients {CONCURRENT_CONNECTIONS}"
+                f" --threads {THREADS}"
+                f" --max-concurrent-streams {REQUESTS_COUNT}"
+                f" --duration {DURATION}"
+            ),
+        },
+    ]
+
+    def test_stress(self):
+        server = self.get_server("deproxy")
+        server.conns_n = 1
+        server.set_response(
+            deproxy.Response.create_simple_response(
+                status="200",
+                headers=[("content-length", "0")],
+                date=deproxy.HttpMessage.date_time_string(),
+            )
+        )
+        self.start_all_services(client=False)
+
+        self.setup_fail_function_test("tdb_alloc_blk", 10, -1, 0, 0)
+        client = self.get_client("h2load")
+
+        self.oops_ignore = ["ERROR"]
+        client.start()
+        self.wait_while_busy(client)
```

Comment:
```suggestion
        self.assertTrue(self.wait_while_busy(client))
```
You should check that the client is finished successfully.